src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,48 @@
+package homework_2.pyramid_printer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
+
+    private static void printPyramid(int levelsOfPyramid) {
+        for (int i = 1; i <= levelsOfPyramid; i++) {
+            for (int j = 1; j <= i; j++) {
+                System.out.print("x");
+            }
+            System.out.println();
+        }
+    }
+
+    public void run() {
+        System.out.println("Please type a positive integer number:");
+        String inputData = readData();
+        int getResult = executeProgram(inputData);
+        printPyramid(getResult);
+    }
+
+    private int executeProgram(String dataFromInput) {
+        int levelsOfPyramid = 0;
+        try {
+            levelsOfPyramid = Integer.parseInt(dataFromInput);
+            if (levelsOfPyramid < 0) {
+                throw new NumberFormatException();
+            }
+        } catch (NumberFormatException e) {
+            System.out.println("You typed not a number or a negative number, please add a positive number");
```

Comment:
you should print "Only 1 non-negative integer is allowed as passed parameter" here and above when input value is less then zero.

Reply:
thank you! I ve seen updates in Teams in HW description, fixed:)